app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -127,5 +127,15 @@ def sync_root_tenant
     def create_cloud_tenant(options)
       CloudTenant.create_cloud_tenant(self, options)
     end
+
+    def self.connected_provider_types
+      types = ManageIQ::Providers::CloudManager.pluck(:type).uniq
+      vcloud_included = types.delete('ManageIQ::Providers::Vmware::CloudManager').present?
```

Comment:
Hrm, I'm not crazy about including a provider-specific check in the base class.  @durandom maybe you have an interesting way of turning this on its head so the base manager doesn't have to ask about vmware specifically.